Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG PREFIX=/usr/local
 RUN dpkg --add-architecture i386
 
 # Add rocm repository
-RUN sh -c 'echo deb [arch=amd64 trusted=yes] http://repo.radeon.com/rocm/apt/5.0.2/ ubuntu main > /etc/apt/sources.list.d/rocm.list'
+RUN sh -c 'echo deb [arch=amd64 trusted=yes] http://repo.radeon.com/rocm/apt/5.3/ ubuntu main > /etc/apt/sources.list.d/rocm.list'
```

Comment:
Dockefile installs `clang-format-10` is that needed?

Reply:
Or is it keeping at `clang-format-10` instead of just `clang-format` ?

Reply:
I've had this conversation with Paul already.  He wants to be explicit with the version.